getting-started/004-place.md

```diff
@@ -1,40 +1,38 @@
-Places
-=======
+# Search an ID to get details on a place
 
-Place search is for when you have an ID and the source it came from and now you just need the details.  
+When you know an identification number and the source it came from, you can use Mapzen Search to get details on the location.  
 
-Once you have [an API key](https://mapzen.com/developers), continue on.
+To get started with a place search, you need a [free, developer API key](https://mapzen.com/developers) and these three pieces of information:
 
-## The Basics
+* source - the data source, such as OpenStreetMap
+* layer - the type of place, such as a venue, address, country.
+* id - the identification number of the item
 
-A place search requires 3 pieces of information:
+If you have all of those, join them together with semicolon and pass them in with the `ids` parameter.  
 
-1. source
-2. layer
-3. id
+For example, this `/place` query looks up the Eiffel Tower in OSM:
 
-Once you have all 3 of those, simply join them together with semicolon and pass in on the ids parameter.  
+http://pelias.bigdev.mapzen.com/v1/place?api_key=pelias-M7dcnto&ids=osm:venue:5013364
 
-For example, this /place query looks up the Eiffel Tower in OSM:
+***From Rhonda -- looks like this returns a radio station in Michigan? Also need to redo the query to include production server and the reader's API key***
```

Comment:
This is known defect for /place that’s actively being worked on.  That request is correct (though the response is wrong) and will work in the near future.